src/CoreShop/Bundle/CoreBundle/Portlet/OrderCartPortlet.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * CoreShop.
+ *
+ * This source file is subject to the GNU General Public License version 3 (GPLv3)
+ * For the full copyright and license information, please view the LICENSE.md and gpl-3.0.txt
+ * files that are distributed with this source code.
+ *
+ * @copyright  Copyright (c) 2015-2017 Dominik Pfaffenbauer (https://www.pfaffenbauer.at)
+ * @license    https://www.coreshop.org/license     GNU General Public License version 3 (GPLv3)
+ */
+
+namespace CoreShop\Bundle\CoreBundle\Portlet;
+
+use Carbon\Carbon;
+use CoreShop\Component\Core\Model\ProductInterface;
+use CoreShop\Component\Core\Model\StoreInterface;
+use CoreShop\Component\Core\Portlet\PortletInterface;
+use CoreShop\Component\Currency\Formatter\MoneyFormatterInterface;
+use CoreShop\Component\Locale\Context\LocaleContextInterface;
+use CoreShop\Component\Pimcore\InheritanceHelper;
+use CoreShop\Component\Resource\Repository\RepositoryInterface;
+use Doctrine\DBAL\Connection;
+use Pimcore\Model\DataObject;
+use Symfony\Component\HttpFoundation\ParameterBag;
+
+class OrderCartPortlet implements PortletInterface
```

Comment:
Guess we could implement the OrderCartPortlet and OrderCartReport in one class implementing both interfaces?